app/src/main/kotlin/com/intive/picover/common/annotation/NoArgConstructor.kt

```diff
@@ -0,0 +1,8 @@
+package com.intive.picover.common.annotation
+
+// Needed, when a class has to define a no-argument constructor.
+// Works configured with no-arg compiler plugin (https://kotlinlang.org/docs/no-arg-plugin.html).
+// Example usage: avoid Firebase DatabaseException, when deserializing object from realtime database to data class.
```

Comment:
Is it a comment or doc?

Reply:
..more like a doc for me :D Adjusted with usage of KDoc syntax :)